src/main/java/uk/gov/hmcts/cmc/claimstore/services/WorkingDayIndicator.java

```diff
@@ -46,4 +48,13 @@ public boolean isPublicHoliday(LocalDate date) {
     public boolean isCustomNonWorkingDay(LocalDate date) {
         return nonWorkingDaysCollection.contains(date);
     }
+
+    public LocalDate getNextWorkingDay(LocalDate date) {
+        requireNonNull(date);
+
+        if(isWorkingDay(date))
+            return date;
+        else
+            return getNextWorkingDay(date.plusDays(1));
```

Comment:
Some formatting issues:
* Space after `if`
* braces around all nested statements (`if (...) { ... } else { ... }`)

Could be compacted to 
```
return isWorkingDay(date) ? date : getNextWorkingDay(date.plusDays(1))
```
 if you think it's readable enough.